src/aether-producer/templates/deployment.yaml

```diff
@@ -125,6 +123,21 @@ spec:
               secretKeyRef:
                 name: {{ .Values.app.secret }}
                 key: admin-password
+          - name: KAFKA_URL
+            value: {{ .Values.app.kafkaUrl }}
+          - name: KAFKA_SU_USER
```

Comment:
we can just make this a value

Reply:
That's fine, I just assumed this was referenced from somewhere since I assume the service url could change.

Reply:
Actually yeah, since it's the CCloud url.